pages/[resourceType]/[id]/evaluate.tsx

```diff
@@ -1,15 +1,27 @@
 import { Center } from "@mantine/core";
 import { useRouter } from "next/router";
+import { useState } from "react";
+import SelectComponent from "../../../components/SelectComponent";
 
 /**
  * Page for evaluate measure functionality.
  * @returns JSX Element
  */
 const EvaluateMeasurePage = () => {
+  const [practitionerValue, setPractitionerValue] = useState("");
   const router = useRouter();
   const { resourceType } = router.query;
   if (resourceType === "Measure") {
-    return <div>Coming soon...</div>;
+    return (
+      <div>
+        <SelectComponent
+          resourceType="Practitioner"
+          setPractitionerValue={setPractitionerValue}
+          practitionerValue={practitionerValue}
+        />
+        <div> {practitionerValue} </div>
```

Comment:
```suggestion
        <div> {practitionerValue} </div>
```
I don't think we need this, since the value should be displayed in the autocomplete